dev/doc/release-process.md

```diff
@@ -2,7 +2,7 @@
 
 ## When the release managers for version `X.X` get nominated ##
 
-- [ ] Create a new issue to track the release process where you can copy-paste the present checklist from `dev/doc/release-process.md`.
+- [ ] Create a new issue to track the release process where you can copy-paste the present checklist from `dev/doc/release-process.md` (we recommend copy/pasting the "For each release" section for X.Y+rc1 and for X.Y.0, removing the useless entries in each copy).
```

Comment:
It took me a while to understand this parenthesis. Maybe this will make it clearer.

```suggestion
- [ ] Create a new issue to track the release process where you can copy-paste the present checklist from `dev/doc/release-process.md` (we recommend duplicating the "For each release" section for X.Y+rc1 and for X.Y.0, removing the useless entries in each copy).
```